web/controllers/auth_controller.ex

```diff
@@ -9,10 +9,19 @@ defmodule RemoteRetro.AuthController do
 
   def callback(conn, %{"code" => code}) do
     user_info = Google.get_user_info!(code)
-    conn = put_session(conn, :current_user, user_info)
-
     user = Repo.get_by(User, email: user_info["email"])
-    user_params = %{email: user_info["email"], google_user_info: user_info, last_login: DateTime.utc_now}
+
+    user_params = %{
```

Comment:
rather than assign all of the attributes of `user_info` into a new data structure one by one, let's use `Map.merge` to produce a new map containing all the attributes of `user_info` along with the new attributes (`google_user_info`, `email`, and `last_login`)